src/defines.h

```diff
@@ -47,6 +48,13 @@
 // #define BATTERY_SHIELD_R1 100 // Board voltage divider resistor Ain to GND in kOhm
 // #define BATTERY_SHIELD_R2 220 // Board voltage divider resistor Ain to INPUT_BOARD in kOhm
 
+//Only works in esp8266
+#if BOARD == BOARD_WEMOSD1MINI
+#define MAX_ODR false // (True) enables Fastest Output Rate (False) caps it to 100Hz (ICM20948 only)
```

Comment:
Why is it supported only on WEMOSD1MINI and not on all ESP8266 or actually all boards? Also, defines.h is for defines not for processing, checks should be done in globals.h